app/plugins/Overview/js/overview.js

```diff
@@ -81,6 +80,8 @@ IPC.on('wallet-update', function(err, result) {
 	} else {
 		updateField(err, 'Locked', '', 'lock');
 	}
+
+	updateField(err, 'Balance: ', unlocked ? bal : '---', 'balance');
```

Comment:
This should go up near where `bal` is initialized. Further, the whole function does nothing when `result === null` so a 
```
if (!result) {
    return;
}
```
at the top would make the code less verbose and more readable.